ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -377,6 +397,65 @@ const Genotype = React.memo(({ variant, individual, isCompoundHet, genesById })
           {filters}
         </small>
       )}
+    </div>
+  )
+}
+
+GenotypeQuality.propTypes = {
+  genotype: PropTypes.object,
+  variant: PropTypes.object,
+  showSampleType: PropTypes.bool,
+}
+
+const Genotype = React.memo(({ variant, individual, isCompoundHet, genesById }) => {
+  const individualGenotypes = variant.genotypes[individual.individualGuid]
+  if (!individualGenotypes) {
+    return null
+  }
+  const genotypes = (Array.isArray(individualGenotypes) ? individualGenotypes : [individualGenotypes]).sort(
+    (a, b) => SAMPLE_TYPE_DISPLAY_ORDER.indexOf(a.sampleType) - SAMPLE_TYPE_DISPLAY_ORDER.indexOf(b.sampleType),
+  )
+
+  if (!isCalled(genotypes[0].cn) && !isCalled(genotypes[0].numAlt)) {
```

Comment:
nitpcik, but shouldn't you check that all the genotypes are not called for this, not just the first one

Reply:
I was thinking that since cn is an SV-only annotation and there will only be one genotype per SV, there would only be one genotype in the list here but I could be wrong.

Reply:
its true for now but if this ever changes its unlikely we would think to go back to this spot in the UI code and fix it. Building the UI to generically work with multiple sample types is better as it means we are unlikely to introduce bugs in the future, as long as it doesn't make the code to complex/ unreadable for now

Reply:
that makes sense, I updated the code to return no call if all genotypes fail that check